hawc/apps/riskofbias/models.py

```diff
@@ -261,18 +261,18 @@ def deactivate(self):
         self.save()
 
     @property
-    def is_complete(self):
+    def is_complete(self) -> bool:
         """
-        The rich text editor used for notes input adds HTML tags even if input
-        is empty, so HTML needs to be stripped out.
+        A review is complete if all scores either:
+        1) have a score not equal to NR (default score when created)
+        2) have an NR score, but notes were added
         """
-        return all(
-            [
-                len(strip_tags(score.notes).strip()) > 0
-                for score in self.scores.all()
-                if score.score not in constants.NA_SCORES
```

Comment:
The logic here has now changed is is consistent with the javascript, which users were already familiar with and therefore takes precedence (and is more relaxed than the prior python rule).